build/release/authors.js

```diff
@@ -0,0 +1,105 @@
+"use strict";
+
+const fs = require( "node:fs" );
+const util = require( "node:util" );
+const exec = util.promisify( require( "node:child_process" ).exec );
+const rnewline = /\r?\n/;
+const rdate = /^\[(\d+)\] /;
+
+const ignore = [
+	/dependabot\[bot\]/
+];
+
+function compareAuthors( a, b ) {
+	const aName = a.replace( rdate, "" ).replace( / <.*>/, "" );
+	const bName = b.replace( rdate, "" ).replace( / <.*>/, "" );
+	return aName === bName;
+}
+
+function uniq( arr ) {
+	const unique = [];
+	for ( const item of arr ) {
+		if ( ignore.some( re => re.test( item ) ) ) {
+			continue;
+		}
+		if ( item && !unique.find( ( e ) => compareAuthors( e, item ) ) ) {
+			unique.push( item );
+		}
+	}
+	return unique;
+}
+
+function cleanupSizzle() {
+	console.log( "Cleaning up..." );
+	return exec( "npx rimraf .sizzle" );
+}
+
+function cloneSizzle() {
+	console.log( "Cloning Sizzle..." );
+	return exec( "git clone https://github.com/jquery/sizzle .sizzle" );
+}
+
+async function getLastAuthor() {
+	const authorsTxt = await fs.promises.readFile( "AUTHORS.txt", "utf8" );
```

Comment:
`fs.promises` is deprecated, the new way to import it is via `node:fs/promises`. I guess the problem is the latter doesn't work on Node 10 but we are not going to run this task on our rusty old Jenkins, are we? Let's use the official API then.

Reply:
I think we talked about that, but had decided to use `fs.promises` for now and then when we drop node 10, do a find/replace on all of them. Then we don't have to keep straight which are used in Node 10 and which aren't.